Sources/Rendering/Core/VolumeMapper/index.d.ts

```diff
@@ -233,6 +238,19 @@ export interface vtkVolumeMapper extends vtkAbstractMapper {
 	update(): void;
 }
 
+/**
+ * create the piecewise and the color transfer functions for the radon blending.
+ * @static
+ * @param {object} values: Store all needed values to create the CTF:
+ * {
+ *   min: Define minimum voxel value
```

Comment:
what does it mean minimum voxel value. Is it the same as `getPointsData().getScalars().getRange()[0]`?
Is it in hounsfield unit ?

Reply:
Description of the method has been improved to explain it.